pandas/tests/plotting/frame/test_frame_color.py

```diff
@@ -15,6 +15,8 @@
     _check_plot_works,
 )
 
+from pandas.plotting._matplotlib.compat import mpl_ge_3_6_0
```

Comment:
I think this needs to be imported in the test

Reply:
thx, moved